diff --git a/app/app_config/config_countries.py b/app/app_config/config_countries.py
index bc71ac9..8d35126 100644
--- a/app/app_config/config_countries.py
+++ b/app/app_config/config_countries.py
@@ -143,6 +143,7 @@ class ConfCountries:
                 "eu.europa.ec.eudi.tax_residency_vc_sd_jwt",
                 "eu.europa.ec.eudi.employee_mdoc",
                 "eu.europa.ec.eudi.pid_mdoc_deferred",
+                "jp.ac.nii.academic_credit",
             ],
         },
         "PT": {
