--- app/app_config/config_countries_local.py
+++ app/app_config/config_countries_local.py
@@ -147,6 +147,8 @@
                 "jp.ac.aaa-university.student_id",
                 "jp.ac.bbb-university.student_id",
                 "jp.ac.ccc-university.student_id",
+                "com.dentsusoken.academic_credit.2024.fall",
+                "com.dentsusoken.academic_credit.2025.spring",
             ],
         },
         "PT": {
