--- app/route_oidc_local.py
+++ app/route_oidc_local.py
@@ -1190,9 +1190,7 @@
         session_manager.add_session(
             session_id=session_id,
             user_data={"extra": extra},
         )

-    credential_issuer = ConfFrontend.registered_frontends[cfgservice.default_frontend][
-        "url"
-    ]
+    credential_issuer = cfgservice.service_url.rstrip("/")

     credential_offer = {
